### PR TITLE
Allow canceling draft invoices

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: l10n_ch_base_bank/invoice.py [hunk 1]
```diff
@@ -103,7 +103,8 @@ def _check_bvr(self, cr, uid, ids, context=None):
         """
         invoices = self.browse(cr, uid, ids)
         for invoice in invoices:
-            if invoice.reference_type == 'bvr' and invoice.state != 'draft':
+            check_state = invoice.state not in ('draft', 'cancel')
```
Comment (yvaucher) [inline l10n_ch_base_bank/invoice.py:106]: Wouldn't it be more readable to skip check, by writing:

```
if invoice.state in ('draft', 'cancel'):
    continue
```

It's certainly only a matter of taste.

**Comment by yvaucher** [comment]
:+1: 

travis pep8 error is unrelated

File: l10n_ch_base_bank/invoice.py (hunk 1 above)
Comment (ghost) [inline l10n_ch_base_bank/invoice.py:106]: Ok I did the change, in my branch, should I submit again the pull request?

Comment (yvaucher) [inline l10n_ch_base_bank/invoice.py:106]: You can update this pull request by updating you branch

**Pull request closed without merging**